### DISCUSSION TITLE
Optimize the image in the README

### DISCUSSION THREAD

**Discussion by olimorris** [frankroeder/parrot.nvim#56] [General]
Firstly, big fan of this plugin. Keep up the great work. Always great to see the LLM plugin community in Neovim thriving.

Heads up...the image in your README is c. 3mb and it takes a while to render...even though my connection is c. 300 mbps.

**Comment by frankroeder** [comment]
Dear @olimorris,
Thank you very much for your feedback. If I weren't focused on doing my own "thing" with `parrot.nvim`, `codecompanion.nvim` would definitely be one of my top Nvim LLM plugins to choose from.
By the way, the quality of your Lua code is a great source of inspiration!

Yes, all the GIFs are way too big. It makes complete sense to replace them now that I know people like you are watching 👀 😄.

**Reply by olimorris** [reply]
Love what you're doing with`parrot.nvim` and keep up the awesome work. It's great we have so many fantastic plugins to learn from.

I had a similar challenge a few months ago with all my GIFs. I then found out you can upload videos (up to 10mb) and I've mostly switched to that (unless I can get a GIF below 1mb).

**Reply by frankroeder** [reply]
Dear @olimorris, thank you again for the suggestion. I now have videos that are almost all below 500KB.
